deployment/cognito.sh

```diff
@@ -45,6 +45,6 @@ aws cognito-idp update-user-pool-client --region $REGION --user-pool-id $cognito
 --supported-identity-providers IDC \
 --allowed-o-auth-flows code \
 --allowed-o-auth-scopes "phone" "email" "openid" "profile" "aws.cognito.signin.user.admin" \
---logout-urls "https://$amplifyDomain/"  \
---callback-urls "https://$amplifyDomain/" \
---allowed-o-auth-flows-user-pool-client
\ No newline at end of file
+--callback-urls "[\"https://$amplifyDomain/\"]" \
```

Comment:
This change makes no sense. The docs of the CLI expect a space separated list of strings for both callback-urls and logout-urls https://docs.aws.amazon.com/cli/latest/reference/cognito-idp/update-user-pool-client.html